hil/api.py

```diff
@@ -1424,3 +1413,19 @@ def _must_find_n(obj_outer, cls_inner, name_inner):
                                     obj_outer.__class__.__name__,
                                     obj_outer.label))
     return obj_inner
+
+
+def _maintain(project, node, node_label):
+    """Perform maintenance tasks."""
```

Comment:
Could use a more detaild docstring here.

Reply:
The checks have been moved.